packages/core/src/control-label/ControlLabel.css

```diff
@@ -64,6 +72,5 @@
 }
 
 .backwardsCompat .uitkControlLabel-label {
-  line-height: var(--uitk-text-base-lineHeight);
```

Comment:
Is this not needed anymore?

Reply:
thats the default anyway for the class (line 35)